upgrade/helpers/tasks.py

```diff
@@ -542,6 +542,42 @@ def upgrade_using_foreman_maintain():
                 os.environ.get('TO_VERSION')))
 
 
+def upgrade_puppet3_to_puppet4():
```

Comment:
What if in future we want to upgrade from 4 to 5. New function ? Or do u think we can make this func portable enough to handle that ? or are we really going to manually upgrade 4 to 5 or its automatic henceforth ?

Reply:
As of now, i don't see any scenario where we need a task to upgrade puppet 4 to 5. If we are upgrading satellite 6.3 to 6.4, it is automatically going to upgrade puppet from 4 to 5.

Reply:
no there is no future 4 > 5 
With 6.4 you land straight on puppet5